### ISSUE TITLE
Totally borked mounts after upgrading Ubuntu to 22.04.1 LTS

### ISSUE THREAD

**Issue by thunderstorm654** [TechPerplexed/Gooby#216]
Hi,

I ran an upgrade this morning but since rebooting I cannot get gooby to start my mounts anymore. System cleanup gets stuck "waiting on mounts to be created". None of my docker apps have come up.

I've tried reinstalling rclone, even to the point of completely removing it and redoing the config so it's all fresh, but this hasn't made any difference. I get an error at the end of install: "Job for mergerfs.service failed because the control process exited with error code."

If I check out "systemctl status mergerfs.service" the following is shown:

`root@ss-ex42 ~ # systemctl status mergerfs.service
× mergerfs.service - MergerFS overlay writable storage and local mount
     Loaded: loaded (/etc/systemd/system/mergerfs.service; enabled; vendor pres>     Active: failed (Result: exit-code) since Sat 2022-10-08 11:48:53 CEST; 3mi>    Process: 2482 ExecStartPre=/bin/mkdir -p ${UPLOADS} ${MEDIA} ${LOCALFILES} >    Process: 2483 ExecStart=/usr/bin/mergerfs -o use_ino,hard_remove,auto_cache>        CPU: 11ms

Oct 08 11:48:53 ss-ex42 systemd[1]: mergerfs.service: Scheduled restart job, re>Oct 08 11:48:53 ss-ex42 systemd[1]: Stopped MergerFS overlay writable storage a>Oct 08 11:48:53 ss-ex42 systemd[1]: mergerfs.service: Start request repeated to>Oct 08 11:48:53 ss-ex42 systemd[1]: mergerfs.service: Failed with result 'exit->Oct 08 11:48:53 ss-ex42 systemd[1]: Failed to start MergerFS overlay writable s>lines 1-12/12 (END)`

I've tried rclean, numerous reboots and to no avail, mergerFS does not want to play ball.

In the /mnt directory I have an rclone folder which appears to be populated with folders, there's a Google one too which is empty. The rclone folder appears to be working as I just added a new folder in Google Drive and it's shown up in the mount. Any ideas? Pulling hair out... And this is before I thought about migrating to omnistream :-)

Many thanks!

**Comment by thunderstorm654** [comment]
Decided to cut my losses and reset my server and switch to Debian 10, as I know this is what you guys are using :). Managed to get back up and running and restored, The only remaining question I have before I proceed to OmniStream is now I've done a new backup of my server (which got a different hostname), it's created two folders in the Backup folder of my Google Drive for it. Most of the files are in one folder but it's decided to put Gooby\Deluge-full.tar.gz in the duplicate server folder. Any ideas why it's doing that?

Cheers!
